tests/shared/nlu/training_data/test_training_data.py

```diff
@@ -64,6 +64,13 @@ def test_wit_data():
     assert td.entities == {"location", "wit$datetime"}
 
 
+def test_watson_data():
```

Comment:
This one fails as you have to convert the training data first.

Reply:
these were part of the old implementation indeed. i removed them.